src/python/queries.rs

```diff
@@ -0,0 +1,205 @@
+use crate::{AmlError, ExpectedAmLabel, Result, FUNC_NAME_CAPTURE};
+use tree_sitter::{Parser, Query};
+use tree_sitter_python::language;
+
+const IMPORT_ALIAS_CAPTURE: &str = "import.alias";
+
+fn new_parser() -> Result<Parser> {
+    let mut parser = Parser::new();
+    parser.set_language(language())?;
+    Ok(parser)
+}
+
+fn get_node_qualname(node: &tree_sitter::Node, source: &str) -> anyhow::Result<String> {
```

Comment:
We should use the library Result (and the AmlError type) for functions in the library part of the crate. That makes it easier for library users downstream to match and choose what they want to do when there's an error. I think that mostly means moving the `AmlError::InvalidText` conversion a few lines down